common/src/app/common/types/page/grid.cljc

```diff
@@ -0,0 +1,44 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+;;
+;; Copyright (c) UXBOX Labs SL
+
+(ns app.common.types.page.grid
+  (:require
+   [app.common.spec :as us]
+   [clojure.spec.alpha :as s]))
+
+;; --- Board grids
+
+(s/def :grid/color string?)
```

Comment:
We should stop use single segment undeclared namespace.
I mean, if we have `color` and `internal/color`, we pobably need to create a specific ns for it with
`(:require [app.common.types.page.grid.internal :as-alias internal])`

And then use `(s/def ::internal/color string?)` for this case.
But having `color` meaning different things, is a bit confusing (I know that is not related to this PR, because it has been done in this way some time ago).